spec/latest/schema.org/index.html

```diff
@@ -446,16 +446,16 @@
           <td>hydra:required</td>
         </tr>
         <tr>
-          <td>readonly</td>
+          <td>readable</td>
```

Comment:
My `grep` missed this :smile:

Reverted; amended as f83cb47042a7cd724b650245f5287589826d8ad7.